.gitignore

```diff
@@ -59,3 +60,6 @@ QA/*
 QSF/*
 SampleApps/*
 *VSIX*
+/SDKExamples.UWP/SDKExamples.UWP.sln.DotSettings.user
```

Comment:
try to handle all DotSetting.user generated files, not one by one. Check for example how it is done for the ".lock" file.